hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/DeleteBlocksCommandHandler.java

```diff
@@ -326,9 +326,13 @@ private void processCmd(DeleteCmdInfo cmd) {
 
       DeletedContainerBlocksSummary summary =
           DeletedContainerBlocksSummary.getFrom(containerBlocks);
-      LOG.info("Start to delete container blocks, TXIDs={}, "
+      if (LOG.isDebugEnabled()) {
```

Comment:
Can we remove the `if (LOG.isDebugEnabled())` and just change the log message to debug level, ie:

```
LOG.debug("Start to delete container blocks, TXIDs={}",
            summary.getTxIDSummary());
```

Inside the logger, it applies the `if (LOG.isDebugEnabled())` so if we just use a debug level log, then we can remove the extra if statement.

Reply:
Unfortunately we're better off using `if (LOG.isDebugEnabled())` in this case.  `getTxIDSummary()` performs quite a bit of stream/string processing, which we would like to avoid if the log is going to be discarded.

https://github.com/apache/ozone/blob/47f472f866be980ae17ca68387da85a0771c03c0/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/DeletedContainerBlocksSummary.java#L93-L98